doc/api/n-api.md

```diff
@@ -2942,6 +3002,36 @@ The native string is copied.
 The JavaScript `string` type is described in
 [Section 6.1.4][] of the ECMAScript Language Specification.
 
+#### `node_api_create_external_string_utf8`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+```c
+napi_status node_api_create_external_string_utf8(napi_env env,
```

Comment:
IMO, due to performance issues, it's probable that other JS engines, not just V8, may not support external utf8 strings. So this API might not be necessary.

Reply:
I dropped this API.